src/content/troubleshooting/_index.en.md

```diff
@@ -27,7 +27,7 @@ This section will contain information about common deployment issues you can run
 #### TBD
 
 ### Connectivity Issues
-Deployment went through successfully but services/pods on one cluster are unable to connect to services on another clusters. This can be due to multuple factors - IP Sec tunnels, IP Table rules, Gateway pods, Routeagent etc.
+Submariner deployment completed successfully but Services/Pods on one cluster are unable to connect to Services on another cluster. This can be due to multiple factors outlined in the following sections.
```

Comment:
This section is not getting rendered in the final output. Can you take a look.

Reply:
That is by design, refer @mangelajo comment. It is commented out and idea is others will add content to it and enable it. Focus of this PR is lighthouse.